ui/raidboss/data/05-shb/raid/e7n.txt

```diff
@@ -4,4 +4,99 @@
 hideall "--Reset--"
 hideall "--sync--"
 
+# -ii 4C30 4C32 4C33 4C36 4C3A 4C49 4D0A 4D0B
+
 0.0 "--Reset--" sync / 21:........:40000010:/ window 10000 jump 0
+
+
+# Opening block
+0.0 "--sync--" sync /:Engage!/ window 0,1
+3.8 "--sync--" sync /:The Idol Of Darkness:4C50:/ window 3,1
+12.8 "Empty Wave" sync /:The Idol Of Darkness:4C52:/ window 12,30
+25.0 "Unshadowed Stake" sync /:The Idol Of Darkness:4C51:/
+37.8 "Words of Motion" sync /:The Idol Of Darkness:4C2B:/
+49.6 "Light's Course" sync /:Unforgiven Idolatry:4C3C:/
```

Comment:
Can you comment out these syncs on Light's Course because their so close to each other?